Library/Homebrew/formula_assertions.rb

```diff
@@ -24,6 +24,12 @@ def assertions
       @assertions ||= T.let(0, T.nilable(Integer))
     end
 
+    sig { params(exp: Object, act: Object, msg: T.nilable(String)).returns(T::Boolean) }
```

Comment:
```suggestion
    sig { params(exp: Object, act: Object, msg: T.nilable(String)).returns(TrueClass) }
```

technically never returns false as it will raise instead.